docs/readme.md

```diff
@@ -0,0 +1,25 @@
+# Taxonomy
+
+## Unclassified Tree
+
+Nodes are deemed *unclassified* when their *scientific name* contains (ignore case) the word `unclassified`; these generate the **Unclassified** tree.
+
+## Environmental Samples Tree
+
+The **Environmental Samples** tree is generated by the nodes whose *scientific name* has prefix (ignore case) `environmental samples`.
+
+# File structure
+
+Each relation is stored as two `csv` files (in and out): first column source, second column target. We have the following relations
```

Comment:
I don't understand the *(in and out)*. Could you add an example of one of the trees specifying its set of files?

Reply:
Ok, now it seems obvious :man_facepalming:, my bad.

And the last question: for `scientific_name` and `rank`, do we also have two CSVs? One file with the node ID in the first conlumn and their `scientific_name` (resp. `rank`) in the second one, and the other file with the `scientific_name` (resp. `rank`) in the first column and all (?) node IDs with that `scientific_name` (resp. `rank`) in the second one?

Reply:
Yes; index as in "in" for "properties".